index.js

```diff
@@ -290,9 +290,9 @@ class Plugin extends MiniAccountsPlugin {
       // TODO: factor reverse-channel lookup into other class?
       await this._store.load('channel:' + channel)
       const accountForChannel = this._store.get('channel:' + channel)
-      if (accountForChannel && channel !== accountForChannel) {
+      if (accountForChannel && account.getAccount() !== accountForChannel._account) {
```

Comment:
Rather than changing this line, the line that sets `this._store.set('channel:' + channel, account)` should be fixed to say `this._store.set('channel:' + channel, account.getAccount())`

Reply:
Even when changing that line you are referring to, this line needs to be fixed as well. `channel !== accountForChannel` is comparing a channel id to an account name.